eth/eventhandler/event_handler_test.go

```diff
@@ -202,6 +202,92 @@ func TestHandleBlockEventsStream(t *testing.T) {
 		}
 	})
 
+	t.Run("test OperatorAdded event fails for same event data", func(t *testing.T) {
```

Comment:
it it possible to check that the exact error is returned?

Reply:
Nope. We can't do it because of how we handle the `MalformedEventError`. 
```go
		if err := eh.handleOperatorAdded(txn, operatorAddedEvent); err != nil {
			eh.metrics.EventProcessingFailed(abiEvent.Name)

			var malformedEventError *MalformedEventError
			if errors.As(err, &malformedEventError) {
				return nil, nil
			}
			return nil, fmt.Errorf("handle OperatorAdded: %w", err)
		}
```